bangazonapi/models/product.py

```diff
@@ -62,7 +62,10 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
+        if len(ratings) != 0:
```

Comment:
This is covered in ticket #15 Can you update this to use a try/except block instead?

Reply:
Updated the logic here to be `try / except ZeroDivisionError`